src/lib/bootloader/grub2_widgets.rb

```diff
@@ -308,7 +308,7 @@ def contents
             HBox(
               HSpacing(2),
               # text entry
-              Password(Id(:pw1), Opt(:hstretch), _("&Password")),
+              Password(Id(:pw1), Opt(:hstretch), _("&Password for GRUB2 User 'root'")),
```

Comment:
NTH: you might also change the comment for translators saying something like: $this and $that labels have to both fit into XX characters (XX < 80) as they will appear next to each other on the screen. If really needed, you might split it to two lines.